e2e-tests/test-app-support_element.spec.js

```diff
@@ -0,0 +1,27 @@
+import { test, expect } from '../playwright';
+
+test('Verify Support Elements', async ({ page }) => {
```

Comment:
Can we update this to something like `should display all support links with correct URLs in Preferences > Support tab`

or even

```javascript
test.describe('Settings > Support', () => {
  test('should display all support links with correct URLs', async ({ page }) => {
    // test code here
  });
});
```